harvesters/helpers.py

```diff
@@ -1,11 +1,20 @@
 import re
+from harvester_adapters.ckan import settings
+from harvesters.logs import logger
 
 
 def clean_tags(tags):
     ret = []
-    pattern = re.compile('[^A-Za-z0-9\s_\-!?]+')
+    pattern = re.compile(r'[^A-Za-z0-9\s_\-!?]+')
```

Comment:
this is throwing a warning when running the tests 

`harvesters/helpers.py:6
  /Users/juliekramer/Projects/ckan-ng-harvester-core/harvesters/helpers.py:6: DeprecationWarning: invalid escape sequence \s
    pattern = re.compile('[^A-Za-z0-9\s_\-!?]+')

-- Docs: https://docs.pytest.org/en/latest/warnings.html`

Reply:
I think this is already fixed by adding this **r**. I didn't see this warning sisnce last update